modules/nextflow/src/main/groovy/nextflow/executor/BashWrapperBuilder.groovy

```diff
@@ -235,7 +235,7 @@ class BashWrapperBuilder {
         // out eval
         for( Map.Entry<String,String> eval : outEvals ) {
             result << "#\n"
-            result <<"nxf_eval_cmd STDOUT STDERR bash -c \"${eval.value}\"\n"
+            result <<"nxf_eval_cmd STDOUT STDERR bash -c \"${eval.value.replace('"','\\\"')}\"\n"
```

Comment:
is it really needed? since the outmost quote are escape, the inner ones are ok.

Reply:
yes, needed, the resulting `.command.sh` snippet needs to be:
```
bash -c "  command with \"double\" quotes   "
```